x-pack/plugins/apm/public/components/fleet_integration/apm_policy_form/settings_definition/tail_sampling_settings.tsx

```diff
@@ -90,6 +91,30 @@ export function getTailSamplingSettings(docsLinks?: string): SettingsRow[] {
           ),
           required: true,
         },
+        {
+          key: 'tail_sampling_storage_limit',
+          type: 'storageSize',
+          label: i18n.translate(
+            'xpack.apm.fleet_integration.settings.tailSampling.tailSamplingStorageLimit',
+            {
+              defaultMessage: 'Tail sampling storage limit',
+            }
+          ),
+          rowTitle: i18n.translate(
+            'xpack.apm.fleet_integration.settings.tailSampling.tailSamplingStorageLimitTitle',
+            { defaultMessage: 'Storage limit' }
+          ),
+          rowDescription: i18n.translate(
+            'xpack.apm.fleet_integration.settings.tailSampling.tailSamplingStorageLimitDescription',
+            {
+              defaultMessage:
+                'The amount of storage space allocated for trace events matching tail sampling policies.',
```

Comment:
Would we want to caution users on setting this value too high. Setting this limit higher than the allowed space (in a container or even a VM) will most certainly cause the APM Server to become unhealthy.

Reply:
Is it possible to fetch that allowed space and include it in this description?

Reply:
That would be the certainly the best, but there isn't a way to do that as users may set limits for their containers, or mount points differently, and we don't necessarily have access to that. There is an open issue on cloud to automatically use the configured disk quota for the server and freeze that so users don't have to tune it.

Reply:
added @elastic/obs-docs for review